2-mandatory/4-stocks.js

```diff
@@ -33,10 +33,25 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Solve the smaller problems, and then build those solutions back up to solve the larger problem.
         Functions can help with this!
 */
+
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+    let sum;
+    let newArr = [];
+    for (let i = 0; i < closingPricesForAllStocks.length; i++){
+        // console.log(closingPricesForAllStocks[i]);
```

Comment:
Rather than commenting out, when you make your final submission, you should just remove these lines altogether.

Reply:
OK